apollo-backend/src/main/java/io/logz/apollo/blockers/BlockerService.java

```diff
@@ -70,6 +71,11 @@ public boolean shouldBlock(Deployment deployment) {
         return false;
     }
 
+    private boolean isUserAllowedToOverride(Deployment deployment, Blocker blocker) {
+        return blockerInjectableCommons.getBlockerDefinitionDao().getOverrideBlockersIdsByUser(deployment.getUserEmail())
+                .stream().anyMatch(id -> id == blocker.getId());
```

Comment:
Why not use some "getOverrideByUserAndBlockerId()"? It would be simpler and more efficient.

Reply:
it retrieve list of integers filtered by user, while not using cache i dont think it will make a difference, and the list will probably be need for the ui